app/models/user.rb

```diff
@@ -1,12 +1,16 @@
 class User < ApplicationRecord
-    has_many :posts, foreign_key: 'author_id'
-    has_many :comments, foreign_key: 'author_id'
-    has_many :likes, foreign_key: 'author_id'
-  
-    validates :name, presence: true
-    validates :posts_counter, numericality: { greater_than_or_equal_to: 0 }
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Kindly add `:confirmable` to the user model and make sure the user can confirm their accounts before logging in, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#:~:text=Ask%20for%20confirmation%20of%20email.) requirement.